.prettierrc

```
@@ -3,5 +3,6 @@
   "semi": false,
   "singleQuote": false,
   "tabWidth": 2,
-  "trailingComma": "es5"
+  "trailingComma": "es5",
+  "//": "temp commit"
```

Let's remove the comment, we don't need it
```suggestion
  "trailingComma": "es5",
```